dqrobotics/solvers/__init__.py

```diff
@@ -8,10 +8,25 @@
     class DQ_QuadprogSolver(DQ_QuadraticProgrammingSolver):
         def __init__(self):
             DQ_QuadraticProgrammingSolver.__init__(self)
+            self.equality_constraints_tolarence = np.finfo(np.float64).eps
```

Comment:
tolerance

Reply:
Sorry, spelled it wrong the fist time then i just somehow convinced myself it is spelled that way afterward....